tests/ES.FX.Microsoft.Data.SqlClient.Tests/Factories/DelegateSqlConnectionFactoryTests.cs

```diff
@@ -0,0 +1,26 @@
+﻿using ES.FX.Microsoft.Data.SqlClient.Factories;
+using Microsoft.Data.SqlClient;
+using Moq;
+
+namespace ES.FX.Ignite.Microsoft.Data.SqlClient.Tests.Factories
+{
+    public class DelegateSqlConnectionFactoryTests
+    {
+        [Fact]
+        public void TestConnectionCreationFactory()
```

Comment:
Better naming please: DelegateFactory_CanCreateConnections
Reason: Test is already implied